--- src/framework/Makefile	2024-07-26 12:02:26
+++ src/framework/Makefile.new	2024-07-26 12:02:16
@@ -124,9 +124,9 @@
 	$(RM) $@ $*.mod
 ifeq "$(GEN_F90)" "true"
 	$(CPP) $(CPPFLAGS) $(CPPINCLUDES) $< > $*.f90
-	$(FC) $(FFLAGS) -c $*.f90 $(FCINCLUDES) -I../external/esmf_time_f90
+	$(FC) $(FFLAGS) -c $*.f90 -I../external/esmf_time_f90 $(FCINCLUDES)
 else
-	$(FC) $(CPPFLAGS) $(FFLAGS) -c $*.F $(CPPINCLUDES) $(FCINCLUDES) -I../external/esmf_time_f90
+	$(FC) $(CPPFLAGS) $(FFLAGS) -c $*.F -I../external/esmf_time_f90 $(CPPINCLUDES) $(FCINCLUDES)
 endif
 
 .c.o:
